drivers/bluetooth/hci/spi.c

```diff
@@ -428,7 +439,7 @@ static int bt_spi_send(struct net_buf *buf)
 
 	/* Buffer needs an additional byte for type */
 	if (buf->len >= SPI_MAX_MSG_LEN) {
-		LOG_ERR("Message too long");
+		LOG_ERR("Message too long (%d)", buf->len);
```

Comment:
`buf->len` is unsigned, so I believe the right format specifier is `%u` and not `%d`